From 3a19d8b8309070f8c14f3a3b4dc4b098ad64190f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Wed, 9 Jun 2021 10:20:57 +0100
Subject: [PATCH 1/2] dtd files are not xml files and shouldn't have xml
 headers

so rpminspect is correct in complaining that they are not valid xml
on inspecting files claiming to be xml

Change-Id: I70379989326c2ea63e6a54b3658ebea4684fa5df
---
 framework/dtd/accelerator.dtd              | 1 -
 framework/dtd/event.dtd                    | 1 -
 framework/dtd/groupuinames.dtd             | 1 -
 framework/dtd/image.dtd                    | 1 -
 framework/dtd/menubar.dtd                  | 1 -
 framework/dtd/statusbar.dtd                | 1 -
 framework/dtd/toolbar.dtd                  | 1 -
 i18npool/source/localedata/data/locale.dtd | 1 -
 officecfg/registry/component-schema.dtd    | 1 -
 officecfg/registry/component-update.dtd    | 1 -
 package/dtd/Manifest.dtd                   | 1 -
 xmerge/source/xmerge/converter.dtd         | 1 -
 xmloff/dtd/office.dtd                      | 1 -
 xmlscript/dtd/dialog.dtd                   | 1 -
 xmlscript/dtd/libraries.dtd                | 2 --
 xmlscript/dtd/library.dtd                  | 2 --
 xmlscript/dtd/module.dtd                   | 2 --
 17 files changed, 20 deletions(-)

diff --git a/framework/dtd/accelerator.dtd b/framework/dtd/accelerator.dtd
index f4dcac65242d..912b652d3cf5 100644
--- a/framework/dtd/accelerator.dtd
+++ b/framework/dtd/accelerator.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/framework/dtd/event.dtd b/framework/dtd/event.dtd
index 7a66d15e571b..5af4a3dd267f 100644
--- a/framework/dtd/event.dtd
+++ b/framework/dtd/event.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/framework/dtd/groupuinames.dtd b/framework/dtd/groupuinames.dtd
index a29b77a4f13e..218939cf9bae 100644
--- a/framework/dtd/groupuinames.dtd
+++ b/framework/dtd/groupuinames.dtd
@@ -24,4 +24,3 @@
 	groupuinames:name CDATA #REQUIRED
 	groupuinames:default-ui-name CDATA #REQUIRED
 >
-
diff --git a/framework/dtd/image.dtd b/framework/dtd/image.dtd
index 59b0f3f92f08..fb69a5e5f4c8 100644
--- a/framework/dtd/image.dtd
+++ b/framework/dtd/image.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/framework/dtd/menubar.dtd b/framework/dtd/menubar.dtd
index b41431f6d0ef..b3469f7324a6 100644
--- a/framework/dtd/menubar.dtd
+++ b/framework/dtd/menubar.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/framework/dtd/statusbar.dtd b/framework/dtd/statusbar.dtd
index 9f4e9e2a35c3..40a5609b7cf6 100644
--- a/framework/dtd/statusbar.dtd
+++ b/framework/dtd/statusbar.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/framework/dtd/toolbar.dtd b/framework/dtd/toolbar.dtd
index 6f2c38260524..7f8ee24e8ab9 100644
--- a/framework/dtd/toolbar.dtd
+++ b/framework/dtd/toolbar.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/i18npool/source/localedata/data/locale.dtd b/i18npool/source/localedata/data/locale.dtd
index 103ab1721c4e..60006d87f439 100644
--- a/i18npool/source/localedata/data/locale.dtd
+++ b/i18npool/source/localedata/data/locale.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/officecfg/registry/component-schema.dtd b/officecfg/registry/component-schema.dtd
index 242b9c6ea513..70bd51bf7d13 100644
--- a/officecfg/registry/component-schema.dtd
+++ b/officecfg/registry/component-schema.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/officecfg/registry/component-update.dtd b/officecfg/registry/component-update.dtd
index d99ee499249e..0659e412305b 100644
--- a/officecfg/registry/component-update.dtd
+++ b/officecfg/registry/component-update.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/package/dtd/Manifest.dtd b/package/dtd/Manifest.dtd
index 2dbdbf00c81d..9e16a007a799 100644
--- a/package/dtd/Manifest.dtd
+++ b/package/dtd/Manifest.dtd
@@ -46,4 +46,3 @@
 	manifest:key-derivation-name CDATA #REQUIRED
 	manifest:salt CDATA #REQUIRED
 	manifest:iteration-count CDATA #REQUIRED>
-
diff --git a/xmerge/source/xmerge/converter.dtd b/xmerge/source/xmerge/converter.dtd
index f7c09f5c2c58..81ecbe69d80e 100644
--- a/xmerge/source/xmerge/converter.dtd
+++ b/xmerge/source/xmerge/converter.dtd
@@ -82,4 +82,3 @@
 <!ELEMENT converter-target       EMPTY>
 
 <!ATTLIST converter-target type  CDATA      #REQUIRED>
-
diff --git a/xmloff/dtd/office.dtd b/xmloff/dtd/office.dtd
index e056d57e5525..5da82955640b 100644
--- a/xmloff/dtd/office.dtd
+++ b/xmloff/dtd/office.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/xmlscript/dtd/dialog.dtd b/xmlscript/dtd/dialog.dtd
index 1b7e6355631f..7290b8d22257 100644
--- a/xmlscript/dtd/dialog.dtd
+++ b/xmlscript/dtd/dialog.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
diff --git a/xmlscript/dtd/libraries.dtd b/xmlscript/dtd/libraries.dtd
index 270d89209bba..3b3b82e535b8 100644
--- a/xmlscript/dtd/libraries.dtd
+++ b/xmlscript/dtd/libraries.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
@@ -33,4 +32,3 @@
 	library:link %boolean; #REQUIRED
 	library:readonly %boolean; #IMPLIED
 >
-
diff --git a/xmlscript/dtd/library.dtd b/xmlscript/dtd/library.dtd
index 4f1512692ad6..c0341ed545b5 100644
--- a/xmlscript/dtd/library.dtd
+++ b/xmlscript/dtd/library.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
@@ -32,4 +31,3 @@
 <!ATTLIST library:element 
 	library:name CDATA #REQUIRED
 >
-
diff --git a/xmlscript/dtd/module.dtd b/xmlscript/dtd/module.dtd
index 5b36ac033d89..b77d62f46498 100644
--- a/xmlscript/dtd/module.dtd
+++ b/xmlscript/dtd/module.dtd
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
  * This file is part of the LibreOffice project.
  *
@@ -23,4 +22,3 @@
 	script:name CDATA #REQUIRED
 	script:language CDATA #REQUIRED
 >
-
-- 
2.31.1

